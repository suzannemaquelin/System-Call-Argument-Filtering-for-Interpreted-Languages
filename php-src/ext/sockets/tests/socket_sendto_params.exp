Warning: socket_sendto(): Length cannot be negative in %s on line %i