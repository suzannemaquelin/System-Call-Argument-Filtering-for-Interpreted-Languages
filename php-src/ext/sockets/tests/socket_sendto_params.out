Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory

Warning: socket_sendto(): Length cannot be negative in /root/php-src/ext/sockets/tests/socket_sendto_params.php on line 4